explainer.md

```diff
@@ -458,6 +458,7 @@ function onSessionStarted(session) {
 Some features recognized by the UA but not explicitly listed in these arrays will be enabled by default for a session. This is only done if the feature does not require a signal of [user intent](privacy-security-explainer.md#user-intent) nor impact performance or the behavior of other features when enabled. At this time, only the following features will be enabled by default:
 | Feature | Circumstances |
 | ------ | ------- |
+| `viewer` | Successfully created any session |
```

Comment:
I probably should have originally phrased this a bit differently.  This (and changes to the next line) should probably read...

>Requested `XRSessionMode` is `inline`, `immersive-vr`, or `immersive-ar`

Reply:
Done!